android-components/components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -531,7 +531,7 @@ class MozillaSocorroService(
 
                 val jsonObject = JSONObject(input)
                 for (key in jsonObject.keys()) {
-                    if (!ignoreKeys.contains(key)) {
+                    if (key.isNullOrEmpty() && !ignoreKeys.contains(key)) {
```

Comment:
```suggestion
                    if (!key.isNullOrEmpty() && !ignoreKeys.contains(key)) {
```
Should this be a "not"? Or I am getting confused?

Reply:
Yes, my mistake.  Should be as you suggested.  Thanks for catching this.